spec/support/metal_controller.rb

```diff
@@ -48,9 +48,11 @@ def google_analytics
   end
 
   def google_tag_manager
-    tracker do |t|
-      t.google_tag_manager :push, { click: 'X', price: 10 }
-      t.google_tag_manager :push, transactionProducts: [{ sku: 'DD44', name: 'T-shirt' }, { sku: 'DD66', name: 'Jeans' }]
+    unless params[:no_events]
```

Comment:
hmmm. it is just the spec. and I thought better than duplicating the action. hmmm...